config/locales/en/prison_views.yml

```diff
@@ -114,7 +114,10 @@ en:
         email_preview: Preview Email
         prisoner_available: Prisoner available
         slot_available: Space available
-        prisoner_not_available: Prisoner unavailable
+        prisoner_banned: Visits banned
+        out_of_vo: No visiting allowance
```

Comment:
Don't think so, VO is Visiting Order. This concept is reference as `vo` in other parts of the code